<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>administrador_tarjeta_justificacion_staf</class>
 <widget class="QWidget" name="administrador_tarjeta_justificacion_staf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>359</width>
     <height>149</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #F4F4F4;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nombre_staff_lbl">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>202</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string>Luis Rolando Banderas Barrera</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="foto_staff_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources.qrc">:/imagenes/Imagenes/imgPrueba.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="visualizar_justificacion_pbtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color: white;
	color: #3E64FF;
	border: 1px solid #3E64FF;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: #3E64FF;
}</string>
   </property>
   <property name="text">
    <string>Visualizar</string>
   </property>
  </widget>
  <widget class="QLabel" name="puesto_lbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string>Puesto</string>
   </property>
  </widget>
  <widget class="QLabel" name="fecha_inicio_lbl">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>71</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string>12-12-2020</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>71</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="fecha_fin_lbl">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>71</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string>14-14-2010</string>
   </property>
  </widget>
  <widget class="QLabel" name="id_permiso_lbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>idPermiso</string>
   </property>
  </widget>
  <zorder>id_permiso_lbl</zorder>
  <zorder>label</zorder>
  <zorder>nombre_staff_lbl</zorder>
  <zorder>foto_staff_lbl</zorder>
  <zorder>visualizar_justificacion_pbtn</zorder>
  <zorder>puesto_lbl</zorder>
  <zorder>fecha_inicio_lbl</zorder>
  <zorder>label_4</zorder>
  <zorder>fecha_fin_lbl</zorder>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
